### ISSUE TITLE
error when executing small, Aborted (core dumped)

### ISSUE THREAD

**Issue by DemonPrince** [PaddlePaddle/DeepSpeech#341]
hello tried to run deepspeech with docker, it downloaded from docker hub, but I could not configure it correctly to only use CPUs.
I would appreciate knowing the complete configuration to only use CPU.

> run_train configuration
`
***********  Configuration Arguments **********
augment_conf_path: conf/augmentation.config 
batch_size: 8
dev_manifest: data/tiny/manifest.tiny
init_model_path: None
is_local: 1
learning_rate: 1e-05
max_duration: 27.0
mean_std_path: data/tiny/mean_std.npz
min_duration: 0.0
num_conv_layers: 2
num_iter_print: 100
num_passes: 20
num_proc_data: 1
num_rnn_layers: 3
output_model_dir: ./checkpoints/tiny
rnn_layer_size: 2048
share_rnn_weights: 1
shuffle_method: batch_shuffle_clipped
specgram_type: linear
test_off: 0
train_manifest: data/tiny/manifest.tiny
trainer_count: 0
use_gpu: 0
use_gru: 0
use_sortagrad: 1
vocab_path: data/tiny/vocab.txt
`

error:
I0717 19:07:04.436843   647 Util.cpp:166] commandline:  --use_gpu=0 --rnn_use_batch=True --log_clipping=True --trainer_count=0 
[INFO 2019-07-17 19:07:04,477 layers.py:2606] output for __conv_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2019-07-17 19:07:04,479 layers.py:3133] output for __batch_norm_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2019-07-17 19:07:04,479 layers.py:7224] output for __scale_sub_region_0__: c = 32, h = 81, w = 54, size = 139968
[INFO 2019-07-17 19:07:04,480 layers.py:2606] output for __conv_1__: c = 32, h = 41, w = 54, size = 70848
[INFO 2019-07-17 19:07:04,482 layers.py:3133] output for __batch_norm_1__: c = 32, h = 41, w = 54, size = 70848
[INFO 2019-07-17 19:07:04,483 layers.py:7224] output for __scale_sub_region_1__: c = 32, h = 41, w = 54, size = 70848
F0717 19:07:04.529284   647 GradientMachine.cpp:67] Unknown model type: nn
*** Check failure stack trace: ***
    @     0x7f9818ed004d  google::LogMessage::Fail()
    @     0x7f9818ed2398  google::LogMessage::SendToLog()
    @     0x7f9818ecfb5b  google::LogMessage::Flush()
    @     0x7f9818ed326e  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f9818b74c0c  paddle::GradientMachine::create()
    @     0x7f9818ea08bb  GradientMachine::createFromPaddleModelPtr()
    @     0x7f9818ea14ce  GradientMachine::createByConfigProtoStr()
    @     0x7f98189883aa  _wrap_GradientMachine_createByConfigProtoStr
    @           0x4c2e1e  PyEval_EvalFrameEx
    @           0x4b9b66  PyEval_EvalCodeEx
    @           0x4c1f56  PyEval_EvalFrameEx
    @           0x4b9b66  PyEval_EvalCodeEx
    @           0x4c17c6  PyEval_EvalFrameEx
    @           0x4b9b66  PyEval_EvalCodeEx
    @           0x4d57a3  (unknown)
    @           0x4eef5e  (unknown)
    @           0x4eeb66  (unknown)
    @           0x4aaafb  (unknown)
    @           0x4c166d  PyEval_EvalFrameEx
    @           0x4b9b66  PyEval_EvalCodeEx
    @           0x4c17c6  PyEval_EvalFrameEx
    @           0x4b9b66  PyEval_EvalCodeEx
    @           0x4c1f56  PyEval_EvalFrameEx
    @           0x4b9b66  PyEval_EvalCodeEx
    @           0x4c1f56  PyEval_EvalFrameEx
    @           0x4b9b66  PyEval_EvalCodeEx
    @           0x4eb69f  (unknown)
    @           0x4e58f2  PyRun_FileExFlags
    @           0x4e41a6  PyRun_SimpleFileExFlags
    @           0x4938ce  Py_Main
    @     0x7f9835098830  __libc_start_main
    @           0x493299  _start
Aborted (core dumped)
Fail in training!

**Comment by ssokhey** [comment]
I think you need to mention some trainer count instead of  `trainer_count: 0`.

If you have 8 core CPU, you can mention `trainer_count` as 8 or whatever suits you (6, 5, 1) but not zero.
